python/xorbits/deploy/__init__.py

```diff
@@ -12,15 +12,118 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from typing import List, Optional, Union
+
+from .._mars.utils import no_default
 from ..core.adapter import mars_new_session, mars_stop_server
 
 
-def init(*args, **kwargs) -> None:
-    mars_new_session(*args, **kwargs)
+def init(
+    address: Optional[str] = None,
+    init_local: bool = no_default,
+    session_id: Optional[str] = None,
+    default: bool = True,
+    timeout: Optional[float] = None,
+    n_worker: int = 1,
+    n_cpu: Union[int, str] = "auto",
+    mem_bytes: Union[int, str] = "auto",
+    cuda_devices: Union[List[int], List[List[int]], str] = "auto",
+    web: Union[bool, str] = "auto",
+    new: bool = True,
+    **kwargs
+) -> None:
+    """
+    Init Xorbits runtime locally or connect to a Xorbits cluster.
+
+    Parameters
+    ----------
+    address: str, optional
+        - if None which is default, address will be "127.0.0.1", a local runtime will be initialized
+        - if specify an address for creating a new local runtime, specify like ``<ip>:<port>``
+        - if connect to a Xorbits cluster address, e.g. ``http://<supervisor_ip>:<supervisor_web_port>``
+    init_local: bool, no default value
+        Indicates if creating a new local runtime.
+
+        - When address is None, ``init_local`` will be True,
+        - Otherwise, if it's not specified, False will be set.
+    session_id: str, optional
+        Session ID, if not specified, a new ID will be auto generated.
+    default: bool
```

Comment:
Since `init` doesn't return anything, the created session must be the default session.